src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java

```diff
@@ -39,7 +40,17 @@ public List<CqlCompilerException> filter() {
       if (filteredCqlTranslatorExceptions.isEmpty()) {
         return Collections.emptyList();
       } else {
-        return filterByLibrary(filteredCqlTranslatorExceptions);
+        List<CqlCompilerException> filteredList = filterByLibrary(filteredCqlTranslatorExceptions);
+        List<CqlCompilerException> newList = new ArrayList<CqlCompilerException>();
```

Comment:
```suggestion
        List<CqlCompilerException> newList = new ArrayList<>();
```

Type declaration is no longer needed on the right hand side. Recommend using [diamond operator](https://www.baeldung.com/java-diamond-operator) instead.